### PR TITLE
End game setting

### LINKED ISSUE #185: Game Loop Bugs
**What are the tasks that need to be done?**
- [x] (Sky) Settings button in Game Over screen needs to open Modal
- [x] (Reeve) Investigate if players can click on people that have been saved by a Medic during the night
- [x] (Reeve) Stop showing "NULL was killed" when no one was killed in the night

### In-Depth Feature Description
Clicking the Settings button in the current Game Over screen crashes the app. Need to hook it up to the Modal component. 
Currently, if a Medic attempts to kill a player but the Medic is able to save them, the statusbar says "NULL was killed" when players wake up.

### Images for Reference:
None, sorry!

### Feature Approval:
- [x]  Feature approved by: @skyn310198

### PR DESCRIPTION
**Pull request by skyn310198**
# Description
This PR fixes #185

Change the Game Over Setting to be able to open Game Setting

# Demo
After: 
image

# Type of change:
* Bug fix (Non-breaking change which fixes a bug)

### CODE REVIEW

**Review by reeved — CHANGES_REQUESTED** [review]
Hi Sky, 

Could you please fix the alignment on your header. They aren't centered vertically. 

It might also be worth looking into using the TopBarSettings component since all screens currently use it. Would make styling the app easier in the future. 

But either of these solutions would work :)

File: frontend/src/Pages/GameOverPage.jsx [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
     });
+
     return (
-        <div>
+        <div> 
             <animated.div style={props}>
-                <TopBarSettings currentScreen={`WINNER are a ${state.winningRole}`} showSettings={true} />
+                <Grid container direction='row' alignItems='center' justifyItems='center'>
+                    <Grid item xs={0} sm={2}></Grid>
+                    <Grid item xs={10} sm={8}><h1>{`WINNER are a ${state.winningRole}`}</h1></Grid>
```
Comment (reeved) [inline frontend/src/Pages/GameOverPage.jsx:27]: Could you please make it say "Winner: ${state.winningRole}!!"

**Review by reeved — CHANGES_REQUESTED** [review]

**Review by Dhors — APPROVED** [review]
Resolved some quick merge conflicts so we could merge your branch into mafia-hands/mafia 
Otherwise, changes look great!

**Review by reeved — APPROVED** [review]
LGTM

**Pull request merged**